src/main/webapp/feed.html

```diff
@@ -38,29 +51,50 @@ <h1>
     <!-- Markdown rendering -->
     <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
 
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAe5HlFZFuhzMimXrKW1z3kglajbdHf_Rc"></script>
+
     <!-- Page scripts -->
     <script type="module">
       import { app } from "/scripts/init.js"
 
+      let map
```

Comment:
Very nitpick but is there a reason to use `let` here as opposed to `var`?

Reply:
They are equivalent as far as I know. When working in js files the linter prefers let for some reason so I was just following that pattern